pygeoapi/api.py

```diff
@@ -1045,9 +1045,7 @@ def describe_collections(self, request: Union[APIRequest, Any],
             if 'format' in collection_data:
                 collection_data_format = collection_data['format']
 
-            is_vector_tile = (collection_data_type == 'tile' and
-                              collection_data_format['name'] not
-                              in [F_PNG, F_JPEG])
+            is_raster_tile = collection_data['name'] == 'WMTSFacade'
```

Comment:
thanks for bringing this up, @simonseyock . Relying on the `collection_data['name'] key` (or the `collection_data_type`, as you were doing before) is problematic when using multiple providers, as it will only take the values of the first provider. So, for instance, if you have a collection that has features, vector tiles, and raster tiles (by this order), it will assume the value for features and it will fail the condition on line 1048. You can test it, using this configuration:
```

        providers:
            - type: feature
              name: GeoJSON
              data: tests/data/ne_110m_lakes.geojson
              id_field: id
            - type: tile
              name: MVT-tippecanoe
              data: tests/data/tiles/ne_110m_lakes
              options:
                bounds: [[-124.953634,-16.536406],[109.929807,66.969298]]
                zoom:
                    min: 0
                    max: 5
              format:
                    name: pbf
                    mimetype: application/vnd.mapbox-vector-tile
            - type: tile
              name: WMTSFacade
              data: https://emotional.byteroad.net/geoserver/gwc/service/wmts
              format:
                  name: png  # png or jpeg
                  mimetype: image/png
              options:
                  wmts_layer: camb:hex350_grid_mental_1920 # the layer name of the wmts
                  wmts_tile_matrix_set: WebMercatorQuad  # the name of the tile matrix set of the wmts.
                  scheme: WebMercatorQuad  # the aligning scheme in pygeoapi.
                  zoom:
                      min: 0
                      max: 20

```